pkg/deployment/config.go

```diff
@@ -175,6 +228,19 @@ func NewConfigFromMap(configMap map[string]string) (*Config, error) {
 			return nil, fmt.Errorf("unsupported %s value: %q", defaultAffinityTypeKey, affinity)
 		}
 	}
+	if err := yaml.Unmarshal([]byte(runtimeClassNames), &nc.RuntimeClassNames); err != nil {
+		return nil, fmt.Errorf("%v cannot be parsed, please check the format", RuntimeClassNameKey)
```

Comment:
we should include the error messages in the error we are returning

Reply:
Good thinking. Updated in fa369bb42